lib/features/registration/widgets/registered_check_animation.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class RegisteredCheckAnimation extends StatefulWidget {
+  @override
+  _RegisteredCheckAnimationState createState() =>
+      _RegisteredCheckAnimationState();
+}
+
+class _RegisteredCheckAnimationState extends State<RegisteredCheckAnimation> {
+  double leftPosition = 0;
+  double topPosition = 0;
+  double imageSize = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.sizeOf(context);
+    return TweenAnimationBuilder(
+      duration: const Duration(milliseconds: 1200),
+      curve: Curves.elasticOut,
+      tween: Tween<double>(begin: 0, end: 1),
+      builder: (context, value, child) {
+        leftPosition = size.width * (0.33 - 0.11 * value);
```

Comment:
perhaps would be better to put these as constants?

Reply:
🤔 leftPosition cannot be const because it uses size; separating 0.33 seems a bit silly to me